src/main/java/bitmap/transformer/Bitmap.java

```diff
@@ -1,30 +1,88 @@
 package bitmap.transformer;
 
+import javax.imageio.ImageIO;
 import javax.xml.xpath.XPath;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.File;
 
 public class Bitmap {
     //instance variable
     // input-file-path output-file-path transform-name
-    String inputFilePath;
-    String outputFilePath;
-    String transformName;
-
-    public Bitmap(String inputFilePath, String outputFilePath, String transformName){
-        this.inputFilePath = inputFilePath;
-        this.outputFilePath = outputFilePath;
-        this.transformName = transformName;
-    }
+    private static BufferedImage image;
+    private static int width;
+    private static int height;
 
-    public void setInputFilePath(String newInputFilePath){
-        this.inputFilePath = newInputFilePath;
+    public Bitmap(BufferedImage image) {
+        this.image = image;
+        width = image.getWidth();
+        height = image.getHeight();
     }
 
-    public void setOutputFilePath(String newOutputFilePath){
-        this.outputFilePath = newOutputFilePath;
+    public void transformGreyScale()

```

Comment:
standard Java syntax has the opening `{` on the same line as the thing it opens:
`public void transofrmGreyScale() {`